caster/apps/gitbash.py

```diff
@@ -9,12 +9,10 @@
 """
 #---------------------------------------------------------------------------
 
-from dragonfly import (Grammar, Mimic, Function)
+from dragonfly import (Grammar, AppContext, Mimic, Key, Text, Function)
```

Comment:
`AppContext`, `Key`, and `Text` should now be imported from `caster.lib.context` and `caster.lib.actions` to support optional use with `Aenea`.

Reply:
Can we put these into another issue? This problem appears to be in many scripts and it'll be tidier to do them all at once, along with any more that need to be switched.

Reply:
You make a fair point. We'll fix it later.